src/playback/view/internal/outputresourceitem.cpp

```diff
@@ -0,0 +1,151 @@
+#include "outputresourceitem.h"
+
+#include <QList>
+
+#include "log.h"
+#include "translation.h"
+
+using namespace mu::playback;
+using namespace mu::audio;
+
+static const QString NO_FX_MENU_ITEM_ID = QString::fromStdString(mu::trc("playback", "No effect"));
```

Comment:
There is a mistake. Translation will not be applied to this string because it is static

Reply:
damn, yeah
probably better to replace it with a getter

Reply:
So a `QT_TRANSLATE_NOOP()` is needed here